### PR TITLE
Wire webapp bumper

### PR DESCRIPTION
**Pull request by arianvp**

### CODE REVIEW

**Comment by arianvp** [comment]
Demonstration that it works:

https://github.com/arianvp/wire-server/pull/3
https://github.com/arianvp/wire-server/runs/2473807029?check_suite_focus=true

File: hack/bin/get-webapp-tag.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+set -eou pipefail
+
+cd $1
+
+git checkout master
+webappversion=$(git describe --tags)
+git checkout $webappversion
+ref=$(git rev-parse --short=6 HEAD)
+# NOTE: the docker tags seem to get the app-config version wrong. But lets duplicate this bug.
+configversion=$(jq -r '.dependencies["wire-web-config-default-staging"]' ./app-config/package.json  | cut -d'#' -f2)
```
Comment (jmatsushita) [inline hack/bin/get-webapp-tag.sh:12]: Do we need to deal with `wire-web-config-default-master` too?

**Review by jmatsushita — APPROVED** [review]
Cool, our own little dependabot in a pipeline :) Can you document the PR a bit explaining:
 - It's triggered every 8 hours
 - It uses wireapp/wire-webapp metadata to deduce the docker tag version
 - It submits a PR to wireapp/wire-server if there's a change in version

File: hack/bin/set-webapp-version.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+git checkout $webappversion
+ref=$(git rev-parse --short=6 HEAD)
+# NOTE: the docker tags seem to get the app-config version wrong. But lets duplicate this bug.
+configversion=$(jq -r '.dependencies["wire-web-config-default-staging"]' ./app-config/package.json  | cut -d'#' -f2)
+
+
+version="$webappversion-$ref-$configversion-production"
+
+
+sed -i "s/  tag: .*/  tag: $version/g" "$CHARTS_DIR/webapp/values.yaml"
```
Comment (jmatsushita) [inline hack/bin/set-webapp-version.sh:21]: Maybe 
```
sed -i "s/  tag: .*/  tag: \"$version\"/g" "$CHARTS_DIR/webapp/values.yaml"
```

https://github.com/arianvp/wire-server/pull/3#discussion_r623706053

File: hack/bin/get-webapp-tag.sh (hunk 1 above)
Comment (arianvp) [reply hack/bin/get-webapp-tag.sh:12]: I wrote a comment in the Backend + Web Team chat.

it seems their tags are a bit buggy

**Comment by lucendio** [comment]
FYI: we decided to move the logic to the respective frontend repositories, but we'll keep the PR open until this is done and to link the changes.

**Comment by lucendio** [comment]
An attempt to solve it on the other end: https://github.com/wireapp/wire-webapp/pull/11158

**Comment by lucendio** [comment]
> An attempt to solve it on the other end: [wireapp/wire-webapp#11158](https://github.com/wireapp/wire-webapp/pull/11158)

The PR has been merged now. Closing this one.

**Comment by lucendio** [comment]
FYI, same is on it way for https://github.com/wireapp/wire-account/pull/2866 and  https://github.com/wireapp/wire-team-settings/pull/4958

**Pull request closed without merging**